docker/UCXPy-MOFED.dockerfile

```diff
@@ -4,18 +4,37 @@ FROM nvidia/cuda:${CUDA_VERSION}-devel-${DISTRIBUTION_VERSION}
 
 # Make available to later build stages
 ARG DISTRIBUTION_VERSION
-
 # Should match host OS OFED version (as reported by ofed_info -s)
 ARG OFED_VERSION=5.3-1.0.5.0
 # Tag to checkout from UCX repository
 ARG UCX_VERSION_TAG=v1.12.1
+# Where to install conda, and what to name the created environment
+ARG CONDA_HOME=/opt/conda
+ARG CONDA_ENV=ucx
+# Name of conda spec file in the current working directory that
+# will be used to build the conda environment.
+ARG CONDA_ENV_SPEC=ucx-py-cuda11.5.yml
+
+ENV CONDA_ENV="${CONDA_ENV}"
+ENV CONDA_HOME="${CONDA_HOME}"
 
+# Where cuda is installed
 ENV CUDA_HOME="/usr/local/cuda"
-ENV PATH="${CUDA_HOME}/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
+
+# This can go once the nvidia images have updated the signing key?
```

Comment:
Not quite sure what to make of this comment. Are current signing keys invalid?

Reply:
https://developer.nvidia.com/blog/updating-the-cuda-linux-gpg-repository-key/ but I realise this is actually not needed (I had forgotten to `docker pull` the base image before testing).